2019/spring/Sterne-TT@9am/pepe-flores.md

```diff
@@ -0,0 +1,17 @@
+
+## Jose Nieves Flores Maynez
+
+1. **Basic** Info:
+	+ Male
+	+ 20 years old
+	+ Born and raised in Mexico
+
+2. _Details_:
+	+ Likes to read
+	+ Enjoys listening to music
+	+ Delights learning and trying different cuisines
```

Comment:
We could change the word Delights for some other structure like 'Has a passion for'  that feels more natural

Reply:
:shipit: